manifests/server/mysqltuner.pp

```diff
@@ -7,58 +7,31 @@
 #   The version to install from the major/MySQLTuner-perl github repository. Must be a valid tag. Defaults to 'v1.3.0'.
 # @param source
 #   Source path for the mysqltuner package.
-# @param environment
-#   Environment variables active during download, e.g. to download via proxies: environment => 'https_proxy=http://proxy.example.com:80'
-#
 class mysql::server::mysqltuner(
   $ensure  = 'present',
   $version = 'v1.3.0',
   $source  = undef,
-  $environment = undef, # environment for staging::file
 ) {
-
+  $tuner_location = '/usr/local/bin/mysqltuner'
```

Comment:
I tend to prefer something like this as a class parameter. This gives some more flexibility to users